3rdparty/libgcov/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+file(GLOB PATCHES patches/*.patch)
+if (PATCHES)
+  foreach (PATCH ${PATCHES})
```

Comment:
Can this be refactored into an utility that we could use to apply patches to future submodules?
E.g: We could make MUSL a submodule and apply patches.

Reply:
Done. Please check `cmake/utilities.cmake`